Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -1,163 +1,137 @@
-import demistomock as demisto
+"""Implementation file for IllumioCore Integration."""
+
 import urllib3
+from illumio import PolicyComputeEngine, ServicePort, convert_protocol
+from illumio.explorer import TrafficQuery
+from illumio.policyobjects import *
```

Comment:
Take this out. Don't use * imports if you can avoid it.

Reply:
Okay